libraries/AP_InertialSensor/AP_InertialSensor_MPU9250.cpp

```diff
@@ -18,6 +18,7 @@
 
 #include <assert.h>
 #include <utility>
+#include <stdio.h>
```

Comment:
I think it is present on all boards now. It wasn't on APM2, but we've stopped supporting that